hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/RocksDBCheckpointDiffer.java

```diff
@@ -188,6 +178,8 @@ public class RocksDBCheckpointDiffer implements AutoCloseable,
   private ColumnFamilyHandle compactionLogTableCFHandle;
   private RocksDB activeRocksDB;
 
+  public static final Set<String> COLUMN_FAMILIES_TO_TRACK_IN_DAG =
+      ImmutableSet.of("keyTable", "directoryTable", "fileTable");
```

Comment:
One suggestion would be to use pre-existing static variables. 
https://github.com/apache/ozone/blob/6805fd4c36b53884fb1feb252389b00b6fcef590/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java#L219

Reply:
Problem with that is cyclic dependency. Currently `columnFamilies` are defined in ozone-manager and `ozone-manager` depends on `rocksdb-checkpoint-differ`. If we use predefined constants from `ozone-manager`, it will create cyclic dependency between `ozone-manager` and `rocksdb-checkpoint-differ`